ProcessMaker/Http/Controllers/Api/ProcessRequestFileController.php

```diff
@@ -86,26 +86,26 @@ class ProcessRequestFileController extends Controller
      * )
      */
     public function index(Request $laravel_request, ProcessRequest $request)
-     {
-		//Retrieve media from ProcessRequest
-		$media = $request->getMedia();
+    {
+        //Retrieve media from ProcessRequest
+        $media = $request->getMedia();
 
-		//Retrieve input variable 'name'
-		$name = $laravel_request->get('name');
+        //Retrieve input variable 'name'
+        $name = $laravel_request->get('name');
 
-		//If no name, retern entire collection; otherwise, filter collection
-		if (! $name) {
-			return new ResourceCollection($media);
-		} else {
-			$filtered = $media->reject(function ($item, $key) use ($name) {
-				if ($item->custom_properties['data_name'] != $name) {
-					return true;
-				}
-			});
+        //If no name, return entire collection; otherwise, filter collection
+        if (!$name) {
+            return new ResourceCollection($media);
```

Comment:
This block looks like a  guard condition that exits the function returning a ne ResourceCollection. So the else part is not necessary. If you do this, the code nesting will diminish and the code will be more readble. For example:
if(!name) {
    return ResourceCollection($media);
}

$filtered = $media....
:
return new ResourceCollection($filtered)